packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3131,7 +3141,30 @@ export class ContainerRuntime
 			const message = `Summary @${summaryRefSeqNum}:${this.deltaManager.minimumSequenceNumber}`;
 			const lastAck = this.summaryCollection.latestAck;
 
-			this.summarizerNode.startSummary(summaryRefSeqNum, summaryNumberLogger);
+			const startSummaryResult = this.summarizerNode.startSummary(
+				summaryRefSeqNum,
+				summaryNumberLogger,
+				latestSummaryRefSeqNum,
+			);
+
+			if (
+				startSummaryResult.invalidNodes > 0 ||
+				startSummaryResult.mismatchNumbers.length > 0
+			) {
+				summaryLogger.sendErrorEvent({
+					eventName: "LatestSummaryRefSeqNumMismatch",
+					...startSummaryResult,
```

Comment:
I'd put this under details, we're trying to reduce the number of ad hoc columns in the output (these will go to a spill-over Payload column in our 1P telemetry database anyway at this point). And it makes it easier to browse and find this info.

```suggestion
					details: startSummaryResult,
```

(it will automatically get `JSON.stringify`'d)